test/jdk/sun/security/pkcs11/PKCS11Test.java

```diff
@@ -825,8 +825,7 @@ private static String fetchNssLib(String osId) {
                 return fetchNssLib(LINUX_X64.class);
 
             case "Linux-aarch64-64":
-                return fetchNssLib(LINUX_AARCH64.class);
-
+                throw new SkippedException("Skipping Linux aarch64 platforms.");
```

Comment:
please add JDK-8319128 either in exception message or as a comment here. My preference would be to add it to the exception message since this is temporary.